v2/app/collegevine-hub/CVHubViewer.jsx

```diff
@@ -1159,7 +1160,13 @@ var PivotViewer = (Pivot.PivotViewer = function (
 
   function outlineItem(item, index, color, ctx, border, lineWidth) {
     var bounds, html
+
     if (item) {
+      if (item != oldHoveredItem){
```

Comment:
Nit: Prefer strict equality in JavaScript.

```suggestion
      if (item !== oldHoveredItem) {
```